x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/components/connector_description_popover.tsx

```diff
@@ -111,55 +116,111 @@ export const ConnectorDescriptionPopover: React.FC<ConnectorDescriptionPopoverPr
         setIsPopoverOpen(false);
       }}
     >
-      <EuiPanel hasBorder={false} hasShadow={false}>
-        {isDisabled && (
-          <EuiFlexGroup>
-            <EuiFlexItem>
-              <EuiCallOut
-                title={i18n.translate(
-                  'xpack.enterpriseSearch.createConnector.connectorDescriptionBadge.notAvailableTitle',
-                  {
-                    defaultMessage:
-                      'This connector is not available as an Elastic-managed Connector',
+      <EuiPanel
+        css={css`
+          max-width: 700px;
+        `}
```

Comment:
Comment: I know we don't typically use custom css but I can't see another way to achieve this with `EuiPanel`. So this is probably fine.